components/ContactsItem.tsx

```diff
@@ -0,0 +1,16 @@
+import { FC } from "react";
+
+interface Props {
+  title: string;
+  text: string;
+}
+
+export const ContactsItem: FC<Props> = ({ title, text, children }) => {
+  return (
+    <div className="contacts-item">
+      <h3 className="item__title">{title}</h3>
+      <p className="item__text">{text}</p>
```

Comment:
`<p className="contacts-item__text">{text}</p>` instead `<p className="item__text">{text}</p>`

because the block is a `contacts-item`